README.md

```diff
@@ -38,7 +38,11 @@ and open source.
 
 - [Add Firebase to your Flutter app](https://firebase.google.com/docs/flutter/setup)
 - [Available plugins](https://firebase.google.com/docs/flutter/setup#available-plugins)
-- [FlutterFire UI](./packages/flutterfire_ui/README.md) (beta)
+- FlutterFire UIs (beta)
+  - [Auth UI](./packages/firebase_ui_auth/README.md)
+  - [Database UI](./packages/firebase_ui_database/README.md)
+  - [Firestore UI](./packages/firebase_ui_firestore/README.md)
+  - [OAuth UI](./packages/firebase_ui_oauth/README.md)
```

Comment:
```suggestion
```

Reply:
this one is rarely used directly and there are notes on `firebase_ui_auth` about `firebase_ui_oauth` installation